doc/manuals/user/walkthrough_apiv1.md

```diff
@@ -319,11 +319,11 @@ The updateContext request payload contains a list of contextElement
 elements. Each contextElement is associated to an entity (whose
```

Comment:
"doc/manuals/user/walkthrough_apiv1.md 100644 → 100755"

It seems that file has change permissions from 644 to 755. I understand this is not on purpose so it should be changed back to 644.